embeddings/pipeline/lightning_pipeline.py

```diff
@@ -27,6 +27,11 @@ class LightningPipeline(
         "mode": "min",
         "patience": 3,
     }
+    DEFAULT_MODEL_CHECKPOINT_KWARGS: Dict[str, Any] = {
+        "filename": "last",
+        "monitor": None,
+        "save_last": False,
+    }
```

Comment:
Do we only want to save the last epoch checkpoint? what if the model will start to overfit heavily?

Reply:
We have early stopping as default in our pipelines